src/strands_tools/bright_data.py

```diff
@@ -0,0 +1,598 @@
+                },
+                "zone": {
+                    "type": "string",
+                    "description": "Bright Data zone to use (optional, overrides default zone)",
+                },
+                "query": {
+                    "type": "string",
+                    "description": "Search query (required for search_engine)",
+                },
+                "engine": {
+                    "type": "string",
+                    "description": "Search engine to use: google, bing, or yandex (default: google)",
+                },
+                "language": {
+                    "type": "string",
+                    "description": "Two-letter language code (hl parameter for Google)",
+                },
+                "country_code": {
+                    "type": "string",
+                    "description": "Two-letter country code (gl parameter for Google)",
+                },
+                "search_type": {
+                    "type": "string",
+                    "description": "Type of search (images, shopping, news, etc.)",
+                },
+                "start": {
+                    "type": "integer",
+                    "description": "Results pagination offset (0=first page, 10=second page)",
+                },
+                "num_results": {
+                    "type": "integer",
+                    "description": "Number of results to return (default 10)",
+                },
+                "location": {
+                    "type": "string",
+                    "description": "Location for search results (uule parameter)",
+                },
+                "device": {
+                    "type": "string",
+                    "description": "Device type (mobile, ios, android, ipad, android_tablet)",
+                },
+                "return_json": {
+                    "type": "boolean",
+                    "description": "Return parsed JSON instead of HTML/Markdown",
+                },
+                "source_type": {
+                    "type": "string",
+                    "description": "Type of data source for web_data_feed"
+                    + "(e.g., 'linkedin_person_profile', 'amazon_product')",
+                },
+                "num_of_reviews": {
+                    "type": "integer",
+                    "description": "Number of reviews to retrieve (only for facebook_company_reviews)",
+                },
+                "timeout": {
+                    "type": "integer",
+                    "description": "Maximum time in seconds to wait for data retrieval (default: 600)",
+                },
+                "polling_interval": {
+                    "type": "integer",
+                    "description": "Time in seconds between polling attempts (default: 1)",
+                },
+            },
+            "required": ["action"],
+            "allOf": [
```

Comment:
OneOf, AnyOf, AllOf statements are not fully supported with all providers, with @tool decorator implementation this will be also covered ^^